### ISSUE TITLE
Will it create problems to prescribe the maximum incoming solar radiation?

### ISSUE THREAD

**Issue by glwagner** [CliMA/ClimaOcean.jl#175]
It seems hard to use the tabulated albedo with just any radiation product, because they each have independent notions about what the incoming radiation should be. I feel that the `PrescribedAtmosphere` should really provide the reference value for the solar constant, rather than the albedo. Otherwise if we use idosyncratic radiation products, we have to remember to simultaneously update `TabulatedAlbedo` --- not consistent with our conviction to be "friendly" / provide an easy-to-use user interface, I think.

https://github.com/CliMA/ClimaOcean.jl/blob/b3b44b709c70b296adb63cf7b2f4552334dcfd1b/src/OceanSeaIceModels/CrossRealmFluxes/tabulated_albedo.jl#L138

**Comment by simone-silvestri** [comment]
I am not sure I understand. Do you mean that different radiation products might have, for example, longwave and shortwave lumped together? Or maybe that longwave and shortwave might be themselves split into different bands? That, indeed, might be a problem with how we define albedo and emissivity. Probably, there should be a "regularization" in the `OceanSeaIceSurfaceFluxes` type which makes sure that the atmosphere is consistent with the radiation and the `SimilarityTheory` type.

**Comment by glwagner** [comment]
Yeah. Imagine trying to simulation a different climate state with a stronger sun. This code makes a lot of assumptions.
